apps/etterlatte-beregning/src/main/kotlin/beregning/BeregningRoutes.kt

```diff
@@ -37,7 +37,7 @@ fun Route.beregning(
             withBehandlingId(behandlingKlient) {
                 logger.info("Henter overstyrBeregning med behandlingId=$it")
 
-                val overstyrBeregning = beregningService.hentOverstyrBeregning(behandlingId, brukerTokenInfo)?.toDTO()
+                val overstyrBeregning = beregningService.hentOverstyrBeregningPaaBehandlingId(behandlingId, brukerTokenInfo)?.toDTO()
```

Comment:
Hva er det med deg og sykt lange funksjonsnavn? 😛 
`hentOverstyrBeregning(behandlingId)` forteller jo implisitt at den henter basert på behandlingId 🤷

Reply:
Joda men det var mer bare for å skille den fra den som heter `hentOverstyrBeregning...`

Reply:
Bare å spinne opp forslag